2765. Longest Alternating Subarray/solution.py

```diff
@@ -11,39 +10,81 @@ def alternatingSubarray(self, nums: List[int]) -> int:
 
     	# declare a subArraySize variable that keeps track of length of 
```

Comment:
In your comments, you should explain what the variable keeps track of, which you did here, but you don't need to mention the name.